### ISSUE TITLE
GPlay:   java.lang.IllegalStateException - com.owncloud.android.datamodel.FileDataStorageManager.createCapabilityInstance [RC2] 

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/android#1334]
java.lang.IllegalStateException
com.owncloud.android.datamodel.FileDataStorageManager.createCapabilityInstance

* Reports this week: 2,344
* Android 7.1 
* Introduced with 2.0.0-rc

```
java.lang.IllegalStateException: 
  at android.database.CursorWindow.nativeGetString (Native Method)
  at android.database.CursorWindow.getString (CursorWindow.java:438)
  at android.database.AbstractWindowedCursor.getString (AbstractWindowedCursor.java:51)
  at android.database.CursorWrapper.getString (CursorWrapper.java:137)
  at com.owncloud.android.datamodel.FileDataStorageManager.createCapabilityInstance (FileDataStorageManager.java:2052)
  at com.owncloud.android.datamodel.FileDataStorageManager.getCapability (FileDataStorageManager.java:1991)
  at com.owncloud.android.utils.ThemeUtils.getCapability (ThemeUtils.java:330)
  at com.owncloud.android.utils.ThemeUtils.primaryColor (ThemeUtils.java:97)
  at com.owncloud.android.utils.ThemeUtils.primaryColor (ThemeUtils.java:93)
  at com.owncloud.android.ui.notifications.NotificationUtils.newNotificationBuilder (NotificationUtils.java:54)
  at com.owncloud.android.files.services.FileUploader.notifyUploadStart (FileUploader.java:1079)
  at com.owncloud.android.files.services.FileUploader.uploadFile (FileUploader.java:1002)
  at com.owncloud.android.files.services.FileUploader$ServiceHandler.handleMessage (FileUploader.java:970)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:154)
  at android.os.HandlerThread.run (HandlerThread.java:61)
```

**Comment by tobiasKaminsky** [comment]
Stracktrace by @jasonbayton shows this as a more detailed message:

> java.lang.IllegalStateException: Couldn't read row 0, col -1 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.

So, again as in #1319 the column does not exists, therefore "-1" and it crashs.

**Comment by jasonbayton** [comment]
Any chance of an APK with the fix please guys?

**Comment by tobiasKaminsky** [comment]
We want to have some other fixes in it and then build a new RC(3).

**Comment by jasonbayton** [comment]
Ok sure. I guess I can sit around without NC on my phone for a few more days...

**Comment by tobiasKaminsky** [comment]
Plan is tomorrow evening, but we need this time to get the pending bugs fixed.

**Comment by tobiasKaminsky** [comment]
Hopefully fixed in #1342

**Comment by AndyScherzinger** [comment]
Should be fixed with 2.0.0RC3, please re-open if is comes up again
